test/java/com/arpnetworking/metrics/portal/reports/impl/chrome/PerOriginConfigsTest.java

```diff
@@ -78,31 +86,14 @@ public void testDispatch() {
         assertFalse(config.isNavigationAllowed("https://other.com/"));
     }
 
-    @Test
-    public void testIsNavigationAllowed() {
```

Comment:
isn't this test still useful?

Reply:
It is! But it's also an accidental copy-paste of [a test for a related component](https://github.com/speezepearson/metrics-portal/blob/permissive/test/java/com/arpnetworking/metrics/portal/reports/impl/chrome/OriginConfigTest.java#L85).